daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md

```diff
@@ -28,146 +34,253 @@ Create a directory for your components file named `components` and inside it cre
 ```
+{{% /codetab %}}
+
+
+{{% codetab %}}
+```bash
+dapr run --app-id orderprocessingservice --app-port 6001 --dapr-http-port 3601 --dapr-grpc-port 60001 --components-path ./components mvn spring-boot:run
+```
+{{% /codetab %}}
+
+
+{{% codetab %}}
+```bash
+dapr run --app-id orderprocessingservice --app-port 6001 --dapr-http-port 3601 --dapr-grpc-port 60001 --components-path ./components python3 OrderProcessingService.py
+```
+{{% /codetab %}}
+
+
+{{% codetab %}}
+```bash
+dapr run --app-id orderprocessingservice --app-port 6001 --dapr-http-port 3601 --dapr-grpc-port 60001 --components-path ./components go run OrderProcessingService.go
+```
+{{% /codetab %}}
+
+
+{{% codetab %}}
+```bash
+dapr run --app-id orderprocessingservice --app-port 6001 --dapr-http-port 3601 --dapr-grpc-port 60001 --components-path ./components npm start
+```
+{{% /codetab %}}
+
+{{< /tabs >}}
 
 And now you can get the secret by calling the Dapr sidecar using the secrets API:
 
 ```bash
-curl http://localhost:3500/v1.0/secrets/my-secrets-store/my-secret
+curl http://localhost:3601/v1.0/secrets/localsecretstore/secret
 ```
 
 For a full API reference, go [here]({{< ref secrets_api.md >}}).
 
 ## Calling the secrets API from your code
 
-Once you have a secret store set up, you can call Dapr to get the secrets from your application code. Here are a few examples in different programming languages:
+Once you have a secret store set up, call Dapr to get the secrets from your application code. Below are code examples that leverage Dapr SDKs for retrieving a secret.
 
-{{< tabs "Go" "Javascript" "Python" "Rust" "C#" "PHP" >}}
+{{< tabs Dotnet Java Python Go Javascript>}}
 
 {{% codetab %}}
-```Go
-import (
-  "fmt"
-  "net/http"
-)
-
-func main() {
-  url := "http://localhost:3500/v1.0/secrets/my-secrets-store/my-secret"
-
-  res, err := http.Get(url)
-  if err != nil {
-    panic(err)
-  }
-  defer res.Body.Close()
+```csharp
 
-  body, _ := ioutil.ReadAll(res.Body)
-  fmt.Println(string(body))
+//dependencies
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Dapr.Client;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading;
+using System.Text.Json;
+
+//code
+namespace EventService
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            string SECRET_STORE_NAME = "localsecretstore";
+            using var client = new DaprClientBuilder().Build();
+            //Using Dapr SDK to get a secret
+            var secret = await client.GetSecretAsync(SECRET_STORE_NAME, "secret");
+            Console.WriteLine($"Result: {string.Join(", ", secret)}");
+            secret = await client.GetSecretAsync(SECRET_STORE_NAME, "secret");
+            Console.WriteLine($"Result for bulk: {string.Join(", ", secret.Keys)}");
+        }
```

Comment:
What is the difference between line 131 and line 133 ?? both seem the same operation.